README.md

```diff
@@ -1,31 +1,30 @@
 # RubocopRunner
 
-[![Gem Version](https://badge.fury.io/rb/rubocop_runner.svg)](https://badge.fury.io/rb/rubocop_runner)
+[![Gem Version](https://badge.fury.io/rb/rubocop_runner.svg)][rubygems]
 
-This gem provides means of running rubocop including auto-correct on all files
-which are currently staged in git. This can be easily used as pre-commit hook
-with the included template.
+This gem provides you with a rake task to install rubocop as a git pre-commit hook.
+If rubocop finds any issues it will abort the commit. It will try to auto-correct
+all issues in the files that are being commited, but will abort the commit even
+if all issues can be auto-corrected so that you can verify the changes before
+actually commiting them.
 
-If rubocop can fix all detected issues itself via autocorrect it will will do
-so. The commit will abort in this situation.
-In case rubocop accepts all code changes the commit continues.
-
-_Although this gem has no tests it's pretty battle tested and is in use at runtastic since years._
+_Although this gem has no tests it's battle tested and is in use at runtastic
+since years._
 
 ## Installation
 
-Add these lines to your application's Gemfile:
```

Comment:
Please keep this statement before the Ruby block.

Reply:
ups